Makefile

```diff
@@ -200,8 +207,8 @@ publish-local: build
 
 .PHONY: validate-no-changes
 validate-no-changes: clean compile docs
-	$(eval UNTRACKED_FILES := $(shell git status --short))
-	$(eval CHANGED_FILES := $(shell git diff --name-only))
+	$(eval UNTRACKED_FILES := $(shell git status --short dist/))
```

Comment:
this filters to just the `dist` directory

Reply:
seems good. but can you help me understand why this filter is necessary?

Reply:
yeah for some reason installing pipenv + all the dependencies is changing the `requirements.txt` file and couldn't figure out why... I'm assuming this was the original intent anyways given the below message says "dist directory not clean"

Reply:
oh wait i think i removed the pip dependency from this step tho so let me try removing